File: Java/Mahasiswa.java
class Mahasiswa extends SivitasAkademik {
    private String nim, fakultas, prodi;

    public Mahasiswa() {
        this.nim = "";
        this.fakultas = "";
        this.prodi = "";
    }

    public Mahasiswa(String nik, String nama, char jenis_kelamin, String asalUniv, String emailUniv, String nim,
            String fakultas, String prodi) {
        super(nik, nama, jenis_kelamin, asalUniv, emailUniv);
        this.nim = nim;
        this.fakultas = fakultas;
        this.prodi = prodi;
    }

    public Mahasiswa(String nim, String fakultas, String prodi) {
        this.nim = nim;
        this.fakultas = fakultas;
        this.prodi = prodi;
    }

    public void setNIM(String nim) {
        this.nim = nim;
    }

    public void setFakultas(String fakultas) {
        this.fakultas = fakultas;
    }

    public void setProdi(String prodi) {
        this.prodi = prodi;
    }

    public String getNIM() {
        return this.nim;
    }

    public String getFakultas() {
        return this.fakultas;
    }

    public String getProdi() {
        return this.prodi;
    }

}

File: Python/Mahasiswa.py
from SivitasAkademik import SivitasAkademik

class Mahasiswa(SivitasAkademik):
    __nim = ""
    __prodi = ""
    __fakultas = ""

    def __init__(self, nik="", nama="", jk='', asalUniv="", emailUniv="", nim = "", fakultas = "", prodi = ""):
        super(Mahasiswa, self).__init__(nik, nama, jk, asalUniv, emailUniv)
        self.__nim = nim
        self.__prodi = prodi
        self.__fakultas = fakultas

    def setNIM(self, nim):
        self.__nim = nim

    def setProdi(self, prodi):
        self.__prodi = prodi

    def setFakultas(self, fakultas):
        self.__fakultas = fakultas

    def getNIM(self):
        return self.__nim
    
    def getProdi(self):
        return self.__prodi
    
    def getFakultas(self):
        return self.__fakultas
    